tests/data/fmi2/README.md

```diff
@@ -1,11 +1,12 @@
 # FMI 2.0 test FMUs
 
 | Name                   | Origin                  | License                                  |
-| ---------------------- | ----------------------- | ---------------------------------------- |
+|------------------------|-------------------------|------------------------------------------|
 | `CraneController.fmu`  | OSP                     | [MPL 2.0](../../../LICENSE)              |
 | `Dahlquist.fmu`        | [Reference FMUs] 0.0.31 | [2-clause BSD](./reference-fmus_LICENSE) |
 | `KnuckleBoomCrane.fmu` | OSP                     | [MPL 2.0](../../../LICENSE)              |
 | `vector.fmu`           | [OSP cpp-fmus]          | [MIT](./osp_cpp-fmus_LICENSE)            |
+| `BouncingBall.fmu`     | [Reference-FMUs] 0.0.37 | [2-clause BSD](./BouncingBall_LICENSE)   |
```

Comment:
If this is one of the reference FMUs, you could just reuse the link (and file) on line 6.

Reply:
Updated tests to use Dahlquist fmu instead